scipy/ndimage/filters.py

```diff
@@ -198,6 +198,29 @@ def convolve1d(input, weights, axis=-1, output=None, mode="reflect",
     return correlate1d(input, weights, axis, output, mode, cval, origin)
 
 
+def _gaussian_kernel1d(sigma, order, radius):
+    """
+    Computes a 1D Gaussian convolution kernel.
+    """
+    if order < 0:
+        raise ValueError('order must be non-hegative')
+    p = numpy.polynomial.polynomial.Polynomial([0, 0, -0.5 / (sigma * sigma)])
+    x = numpy.arange(-radius, radius + 1)
+    phi_x = numpy.exp(p(x), dtype=numpy.double)
+    # Normalizing by the kernel sum for backwards compatibility. Should
+    # this perhaps be phi /= sigma * sqrt(2 * pi) instead?
```

Comment:
I guess it does, and we couldn't break backwards compatibility either, so why even worry?